jax/_src/numpy/lax_numpy.py

```diff
@@ -4733,9 +4733,33 @@ def array_equiv(a1: ArrayLike, a2: ArrayLike) -> Array:
 
 # General np.from* style functions mostly delegate to numpy.
 
-@util.implements(np.frombuffer)
 def frombuffer(buffer: bytes | Any, dtype: DTypeLike = float,
                count: int = -1, offset: int = 0) -> Array:
+  """Convert a buffer into a 1-D JAX array.
+
+  JAX implementation of :func:`numpy.frombuffer`.
+
+  Args:
+    buffer: an object containing the data.
+    dtype: optional. Desired data type for the array. Default is ``float``.
```

Comment:
I think the default here is `float64` since we delegate directly to NumPy, but then the result may be down-cast to `float32` depending on the state of the X64 flag. This is a bit surprising and maybe we should revisit the API, but I think it would be worth saying this.

Reply:
Yes! description added.